apps/chromealive-core/index.ts

```diff
@@ -140,11 +160,10 @@ export default class ChromeAliveCore {
     this.sendEvent('Session.active', sessionObserver.toEvent());
   }
 
-  private static sendOutput(heroSessionId: string) {
+  private static sendDatabox(heroSessionId: string) {
```

Comment:
I find sendDatabox to be a bit too broad of a name as it's not actually sending a `Databox`.